### ISSUE TITLE
can not import named getter/setter functions

### ISSUE THREAD

**Issue by zeroasterisk** [benmosher/eslint-plugin-import#833]
```js
// example.js
import { alias } from './example.export.js';

// example.export.js
export default {
  name: 'a',
  get alias() { return this.name; },
  set alias(name) { this.name = name; },
};
```

> alias not found in './example.export.js' (import/named)

**Comment by ljharb** [comment]
It's a very very bad idea to export getters and setters like that - both because it *will absolutely* break when node ships ES modules, and because getters and setters should always be avoided. Just make it a normal function.

**Comment by zeroasterisk** [comment]
I agree - sadly it's not my code, I'm playing with some `react-native` + `native-base` stuff and they are doing that... I can re-write it, but I came across it and thought I'd bring it up.

**Comment by ljharb** [comment]
The interop between CJS and ESM isn't quite solidified yet.

If you do `import obj from './path'; const { alias } = obj` i would expect it to work, but exporting getters and setters *specifically* gets tricky.

**Comment by zeroasterisk** [comment]
good enough - thanks for looking into it

**Comment by shah-azman-sakin-alam** [comment]
sorry i realised how much of a old issue it is
